### PR TITLE
Improve installation instruction and add homebrew installation instruction

### PR DESCRIPTION
**Pull request by imbsky**

### CODE REVIEW

**Comment by imbsky** [comment]
I squashed commits.

**Comment by tkuchiki** [comment]
Thank you for your Pull Request!
I will confirm later.

**Comment by imbsky** [comment]
Added alp to homebrew by https://github.com/Homebrew/homebrew-core/pull/44060. So now we can install it with homebrew.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,22 @@ alp is Access Log Profiler
...
 # Installation
 
-Download from https://github.com/tkuchiki/alp/releases
+### Binary distribution
+
+You can just pick your download [here](https://github.com/tkuchiki/alp/releases), put it in your PATH as alp, and set it as executable, e.g.
+
+```bash
+sudo cp <downloaded file> /usr/local/bin/alp
+sudo chmod a+x /usr/local/bin/alp
```
Comment (tkuchiki) [inline README.md]: I think that it is better for the following command:
Could you let me know what you think about it?

```diff
- sudo cp <downloaded file> /usr/local/bin/alp
- sudo chmod a+x /usr/local/bin/alp
+ sudo install <downloaded file> /usr/local/bin/alp
```

File: README.md [hunk 2]
```diff
@@ -6,7 +6,22 @@ alp is Access Log Profiler
 
 # Installation
 
-Download from https://github.com/tkuchiki/alp/releases
+### Binary distribution
+
+You can just pick your download [here](https://github.com/tkuchiki/alp/releases), put it in your PATH as alp, and set it as executable, e.g.
```
Comment (tkuchiki) [inline README.md]: Grammarly says `Consider removing it`, so remove it :)

```suggestion
You can pick your download [here](https://github.com/tkuchiki/alp/releases), put it in your PATH as alp, and set it as executable, e.g.
```

File: README.md (hunk 1 above)
Comment (imbsky) [reply README.md]: It makes sense.

File: README.ja.md [hunk 3]
```diff
@@ -5,7 +5,22 @@ alp はアクセスログ解析ツールです。
...
 # インストール
 
-https://github.com/tkuchiki/alp/releases から環境にあったバイナリが含まれる zip ファイルをダウンロードして、解凍してください。
+### バイナリ配布
+
+[ここ](https://github.com/tkuchiki/alp/releases)から任意のOS向けのバイナリをダウンロードすることができ、それをalpとして配置し、このように実行可能なように設定します。
+
+```bash
+sudo cp <ダウンロードしたファイル> /usr/local/bin/alp
+sudo chmod a+x /usr/local/bin/alp
```
[the remark this answers is not in the document]
Comment (imbsky) [reply README.ja.md]: Same as `README.md`.

**Comment by tkuchiki** [comment]
> Added alp to homebrew by Homebrew/homebrew-core#44060. So now we can install it with homebrew.

Thank you so much!

File: README.md (hunk 2 above)
Comment (imbsky) [reply README.md]: Both sentences are grammatically correct, but I'm okay with that!

**Comment by imbsky** [comment]
I will update README asap.

**Comment by imbsky** [comment]
I’m happy to help 😄

**Pull request merged**